plugins/secrets/kv/README.md

```diff
@@ -0,0 +1,125 @@
+# Vault Plugin: Key-Value Secrets Backend [![Build Status](https://travis-ci.org/hashicorp/vault-plugin-secrets-kv.svg?branch=master)](https://travis-ci.org/hashicorp/vault-plugin-secrets-kv)
```

Comment:
@alrs Not sure which file to comment on, but could we move this tree to `builtin/logical/kv` rather than `plugins/secrets/kv`? That's keeping in line with the other built-in plugin locations (PKI, ...).

Reply:
I'm happy to move it to wherever there is consensus.

Reply:
@alrs This does need to be at `builtin/logical/kv`, with `cmd/kv` as a subpackage under that.

See:

https://github.com/openbao/openbao/blob/04f1f227b9e65575385218a9d5d883ad13bd2be7/vault/testing.go#L1686

which in turn expects stuff under `builtin/logical`, e.g.:

https://github.com/openbao/openbao/blob/04f1f227b9e65575385218a9d5d883ad13bd2be7/helper/testhelpers/pluginhelpers/pluginhelpers.go#L45-L50

So swapping `consul` to `kv` won't work (if we remove `consul`), with `kv` here.